src/js/components/SchemaForm.js

```diff
@@ -99,12 +99,14 @@ class SchemaForm extends mixin(StoreMixin, InternalStorageMixin) {
     // Default method.
   }
 
-  handleAddRow(prop, definition, newDefinition) {
+  handleAddRow(prop, definition, newDefinition, index) {
     let propID = Util.uniqueID(prop);
     newDefinition = FormUtil.getMultipleFieldDefinition(
       prop,
       propID,
-      newDefinition
+      newDefinition,
+      undefined,
+      index
```

Comment:
Yeah, let's wait with that to the refactor. I don't want to mess around more with this than I actually have to